config/environments/development.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.configure do
+  Rails.application.routes.default_url_options[:host] = 'localhost:3000'
```

Comment:
https://stackoverflow.com/questions/5691727/how-to-set-config-action-controller-default-url-options-host-on-per